ostd/libs/ostd-macros/src/lib.rs

```diff
@@ -11,11 +11,13 @@ use syn::{parse_macro_input, Expr, Ident, ItemFn};
 ///
 /// # Example
 ///
-/// ```norun
+/// ```compile_fail
+/// #![no_std]
+///
 /// use ostd::prelude::*;
 ///
 /// #[ostd::main]
-/// pub fn main() {
+/// pub fn kmain() {
```

Comment:
The method has been renamed back as `main`. I renamed as `kmain` because clippy warns the `main` should not be used in doctest. But mark it as ignore will fix the warning.